.eslintrc.js

```diff
@@ -1,7 +1,83 @@
+'use strict';
+
 module.exports = {
   root: true,
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'ember'
+  ],
   extends: [
-    'simplabs',
-    'simplabs/plugins/ember',
+    'eslint:recommended',
+    'plugin:ember/recommended'
   ],
+  env: {
+    browser: true
+  },
+  rules: {
+    // TODO this should no longer be necessary with our Ember support matrix
+    'ember/no-get': 'off',
```

Comment:
Can this be removed then?

Reply:
only once we remove the problematic parts of the code, but I wanted to do that in a dedicated PR

Reply:
ok